### PR TITLE
Add blog post about spaCy integration

### PR DESCRIPTION
**Pull request by osanseviero**
Here are screenshots of the different sections

Screen Shot 2021-07-13 at 10 57 51 AM
Screen Shot 2021-07-13 at 10 56 30 AM
Screen Shot 2021-07-13 at 11 01 03 AM
Screen Shot 2021-07-13 at 10 56 38 AM

### CODE REVIEW

File: spacy-in-the-hub.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+![widget](assets/23_spacy/widget.jpeg)
+
+
+### Using existing models
+
+All models from the Hub can be directly installed using `pip install`. 
+
+
+```bash
+pip install https://huggingface.co/spacy/ca_core_news_sm/resolve/main/ca_core_news_sm-any-py3-none-any.whl
```
Comment (ines) [inline spacy-in-the-hub.md:55]: Maybe we could use the `en_core_web_sm` model here and in the example below? Nothing against Catalan, but many people who just copy-paste the code from the blog post probably don't know it and won't be able to play with it immediately 😅

File: spacy-in-the-hub.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
+---
+title: "Welcome spaCy to the Hugging Face Hub"
```
Comment (julien-c) [inline spacy-in-the-hub.md:2]: Add a nice `thumbnail` if possible to optimize social sharing

File: spacy-in-the-hub.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    <a href="/ines">
+        <img class="avatar avatar-user" src="https://aeiljuispo.cloudimg.io/v7/https://s3.amazonaws.com/moonup/production/uploads/1621945859267-noauth.jpeg?w=200&h=200&f=face" title="Gravatar">
+        <div class="bfc">
+            <code>ines</code>
+            <span class="fullname">Ines Montani</span>
+        </div>
+    </a>
+</div>
+
+[spaCy](https://github.com/explosion/spaCy) is a cool library in the Natural Language Processing ecosystem used by dozens of thousands of persons. spaCy allows to easily use models for tasks such as Part of Speech and Named Entities Recognition.
```
Comment (ines) [inline spacy-in-the-hub.md:33]: I think this sounds a bit better as an introduction – "dozens of thousands of" reads a bit weird to me and I've also included some more specifics on what the library does. Does this sound alright to you?

```suggestion
[spaCy](https://github.com/explosion/spaCy) is a popular library for advanced Natural Language Processing used widely across industry. spaCy makes it easy to use and train pipelines for tasks like named entity recognition, text classification, part of speech tagging and more, and lets you build powerful applications to process and analyze large volumes of text.
```

File: spacy-in-the-hub.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+![snippet](assets/23_spacy/snippet.png)
+![snippet](assets/23_spacy/snippet2.png)
+
+You can even make HTTP requests to call the models from the Inference API, which is useful in production settings.
+
+
+### Sharing your models
+
+But probably the coolest feature is that now you can very easily share your models with the `spacy-huggingface-hub` [library](https://github.com/explosion/spacy-huggingface-hub), which extends the `spaCy` CLI with a new command, `push`.
```
Comment (ines) [inline spacy-in-the-hub.md:78]:
```suggestion
But probably the coolest feature is that now you can very easily share your models with the `spacy-huggingface-hub` [library](https://github.com/explosion/spacy-huggingface-hub), which extends the `spaCy` CLI with a new command, `huggingface-hub push`. 
```

File: spacy-in-the-hub.md (hunk 2 above)
Comment (ines) [reply spacy-in-the-hub.md:2]: Can you make something cute with your logo and the spaCy logo maybe? 😊

File: spacy-in-the-hub.md (hunk 3 above)
Comment (osanseviero) [reply spacy-in-the-hub.md]: Thanks a lot, indeed this sounds better

File: spacy-in-the-hub.md (hunk 1 above)
Comment (osanseviero) [reply spacy-in-the-hub.md]: Sounds good! Changed

File: spacy-in-the-hub.md (hunk 2 above)
Comment (osanseviero) [reply spacy-in-the-hub.md:2]: I was thinking of a HF x spaCy thumbnail as we have for other collaborations, but maybe @gary149 has cooler ideas

File: spacy-in-the-hub.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+import en_core_web_sm
+nlp = en_core_web_sm.load()
+```
+
+When you open a repository, you can click `Use in spaCy` and you will be given a working snippet that you can use to install and load the model!
+
+![snippet](assets/23_spacy/snippet.png)
+![snippet](assets/23_spacy/snippet2.png)
+
+You can even make HTTP requests to call the models from the Inference API, which is useful in production settings.
```
Comment (LysandreJik) [inline spacy-in-the-hub.md:73]: How about showing an example of a cURL request? It should be simple enough given that the token isn't necessary for simple requests:

```
curl -X POST  --data '{"inputs": "Hello, this is Omar"}' https://api-inference.huggingface.co/models/spacy/en_core_web_sm
```
```
[{"entity_group":"PERSON","word":"Omar","start":15,"end":19,"score":1.0}]
```

**Review by LysandreJik — APPROVED** [review]
Looks good!

File: spacy-in-the-hub.md (hunk 2 above)
Comment (osanseviero) [reply spacy-in-the-hub.md:2]: Done!

**Pull request merged**